qa/sys/centos/bootstrap.sh

```diff
@@ -2,11 +2,3 @@
 
 yum update
 yum install -y java-1.8.0-openjdk-devel.x86_64
-
-##
-# Install logstash manually from a URL
-##
-BRANCH=${LOGSTASH_BRANCH:-'master'}
```

Comment:
>For you download(if needed) + install should happen in rspec

For me I though it was simplier, everything for this current test is in the same file, maybe I don't understand the advantages of doing that a bootstrap time.

But I don't think this should be a blocker for this current PR, since there is no tests for the upgrade scenario.
I will revert theses lines and let you go ahead with your solution.